### PR TITLE
Updated to reflect bitbuckets url change

### PR DESCRIPTION
**Pull request by TWith2Sugars**

### CODE REVIEW

**Comment by vasily-kirichenko** [comment]
@rojepp I know you use BitBucket a lot, it would be great if you test this change.

**Comment by TWith2Sugars** [comment]
Bitbucket is a bit flaky today, took a few tries to test this my self.

-----Original Message-----

**Comment by rojepp** [comment]
@TWith2Sugars How did you test this? You'd need to use a lib that lives at bitbucket and has source links, right?

File: src/FSharpVSPowerTools.Logic/GoToDefinitionFilter.fs [hunk 1]
```diff
@@ -355,7 +355,8 @@ type GoToDefinitionFilter(textDocument: ITextDocument,
                         | String.StartsWith "https://raw.github.com" _-> formattedGithubUrl
                         | String.StartsWith "https://github.com" _-> formattedGithubUrl
                         | String.StartsWith "https://bitbucket.org" _-> 
-                            sprintf "%s#cl-%d" (url |> replace "/raw/" "/src/") r.StartLine
+                            let fileName = Path.GetFileName r.FileName
+                            sprintf "%s?fileviewer=file-view-default#%s-%d" (url |> replace "/raw/" "/src/") fileName r.StartLine
```
Comment (rojepp) [inline src/FSharpVSPowerTools.Logic/GoToDefinitionFilter.fs]: Bitbucket source urls with line number looks like this: 
> https://bitbucket.org/IntelliFactory/websharper.google.maps/src/45a6136882edfecd168e0c43c17eced387890be4/build.cmd?at=default&fileviewer=file-view-default#build.cmd-7

This looks like what the code does to me. Except the branch is missing (`?at=default`). I haven't tested the code yet.

**Comment by TWith2Sugars** [comment]
Yeah the company has private repos with sourcelink packages. Just tested with that.

-----Original Message-----

**Comment by vasily-kirichenko** [comment]
@TWith2Sugars thanks
@rojepp I see you have not found any source linked repos :)

File: src/FSharpVSPowerTools.Logic/GoToDefinitionFilter.fs (hunk 1 above)
Comment (TWith2Sugars) [inline src/FSharpVSPowerTools.Logic/GoToDefinitionFilter.fs]: Doesn't need the branch as has the commit hash.

**Comment by rojepp** [comment]
I think this is ready to merge.

**Comment by TWith2Sugars** [comment]
One thing I don't get, why do we perform a http request to check if the url exists?

Wouldn't removing that and just letting the browser try to display it be better? If

**Comment by vasily-kirichenko** [comment]
@TWith2Sugars maybe we check the URL to give better feedback?

**Comment by TWith2Sugars** [comment]
All it does is prevent the browser from opening it, if the 3rd part host is down it feels like this feature isn't working. I've noticed this with bitbucket today. There is a little info status bar but it's easily missed.

I've submitted another pr to remove this is if people think it's right: https://github.com/fsprojects/VisualFSharpPowerTools/pull/1199

**Pull request merged**